frontend/src/app/projects/project-detail/training-details/training-details.component.html

```diff
@@ -0,0 +1,119 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div class="w-full">
+  <div class="mb-2 flex h-9 items-center">
+    <h2 class="text-xl font-medium">Training Details</h2>
+    @if (
+      projectUserService.verifyRole("manager") &&
+      (modelService.models$ | async)?.length === 0
+    ) {
+      <a
+        mat-stroked-button
+        class="!ml-4"
+        [routerLink]="
+          (projectService.project$ | async)?.is_archived
+            ? null
+            : [
+                '/project',
+                (projectService.project$ | async)?.slug,
+                'models',
+                'create',
+              ]
+        "
+        [matTooltip]="
+          (projectService.project$ | async)?.is_archived
+            ? 'You cannot add a model to an archived project'
+            : 'Add training details'
+        "
+        [disabled]="(projectService.project$ | async)?.is_archived"
+        ><div class="flex items-center">
+          <mat-icon id="add-icon">add</mat-icon>
+        </div>
+      </a>
+    }
+  </div>
+  <div class="flex flex-col gap-2">
+    @if ((modelService.models$ | async) === undefined) {
+      <ngx-skeleton-loader
+        appearance="circle"
+        [theme]="{
+          'border-radius': '5px',
+          height: '544px',
+          width: '100%',
+          margin: '0px',
+          padding: '0px',
+          border: '1px solid white',
+        }"
+      ></ngx-skeleton-loader>
+    } @else {
+      @for (model of modelService.models$ | async; track model.id) {
+        <div class="collab-card flex w-full select-none flex-col !p-0">
+          <div class="m-3 flex gap-2">
+            <div class="w-fit rounded border bg-blue-100 p-2 text-sm">
+              {{ model.tool.name }} ({{ model.version?.name }})
+            </div>
+            @if (getPrimaryGitModel(model); as gitModel) {
+              <div class="w-fit rounded border bg-blue-100 p-2 text-sm">
+                Revision {{ gitModel.revision }}
+              </div>
+            }
+          </div>
+
+          <div class="flex grow flex-col justify-between">
+            <div class="mx-2.5 mt-2.5 h-card select-text overflow-y-auto">
+              <markdown
+                class="[&_a]:text-blue-900 [&_a]:!underline [&_h1]:text-2xl [&_h2]:text-lg [&_ol]:ml-5
+                  [&_ol]:list-decimal [&_pre]:my-1 [&_pre]:rounded [&_pre]:border [&_pre]:bg-white
+                  [&_pre]:p-2 [&_ul]:ml-5 [&_ul]:list-disc"
+                [data]="readmes.get(model.slug)"
+              ></markdown>
```

Comment:
It might make sense to show some sort of indicator/warning if there is no readme

Reply:
Added a warning, comparable to the model complexity badge (check the stories to see how it looks)